test/integration/buildclient_test.go

```diff
@@ -195,13 +196,14 @@ func NewTestBuildOpenshift(t *testing.T) *testBuildOpenshift {
 
 	openshift.whPrefix = osPrefix + "/buildConfigHooks/"
 	osMux.Handle(openshift.whPrefix, http.StripPrefix(openshift.whPrefix,
-		webhook.NewController(clientWebhookInterface{osClient}, map[string]webhook.Plugin{
+		webhook.NewController(builder.OSClientBuildConfigClient{osClient}, builder.OSClientBuildClient{osClient}, map[string]webhook.Plugin{
```

Comment:
It would be more effective to implement a "real" implementation of ClientWebhookInterface that wraps a client in webhook.